services/gateway_manager/src/main/java/com/futurewei/alcor/gatewaymanager/entity/GatewayWebJson.java

```diff
@@ -0,0 +1,18 @@
+package com.futurewei.alcor.gatewaymanager.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.web.entity.gateway.GatewayEntity;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class GatewayWebJson {
+
```

Comment:
If this class is the return for "list all available gateway", it's better rename it "GatewaysWebJson"

Reply:
ok